priv/www/js/tmpl/consumers.ejs

```diff
@@ -31,7 +31,7 @@
         <td class="c"><%= fmt_boolean(consumer.ack_required) %></td>
         <td class="c"><%= fmt_boolean(consumer.exclusive) %></td>
         <td class="c"><%= consumer.prefetch_count %></td>
-        <td class="c"><%= fmt_boolean(consumer.single_active) %></td>
+        <td class="c"><%= fmt_boolean(consumer.active) %></td>
```

Comment:
Can we also have the activity status here? Since we already have it somewhere.

Reply:
To be implementation specific, metrics fields are stored in Mnesia when some event is propagated, whereas fields in the CLI are not stored anywhere, they are computed on the fly. This is no big deal to add though, especially if it's useful in the channel page.